cores/esp8266/Schedule.cpp

```diff
@@ -60,19 +75,45 @@ bool schedule_function(std::function<void(void)> fn)
         sLast->mNext = item;
     }
     sLast = item;
+
+    if (repeat_us)
+        item->callNow.reset(repeat_us);
```

Comment:
repeat_us is uint32_t. The polledTimeout class used is FastUs. Shouldn't there be a sanity check against the max possible timeout value?